docs/documentation/upgrading/topics/changes/changes-25_0_0.adoc

```diff
@@ -91,6 +91,18 @@ A new `Session State (session_state)` mapper is also included and can be assigne
 
 If an old version of the JS adapter is used, the `Session State (session_state)` mapper should also be used via client scopes as described above.
 
+= `sub` claim is added to access token via protocol mapper
+
+The `sub` claim, which was always added to access token, is now added by default but using a new `Subject (sub)` protocol mapper.
+
+The `Subject (sub)` mapper is configured by default in the `basic` client scope so no extra configuration is required after upgrading to this version.
+
+Only in the case you are using `Pairwise subject identifier` mapper to map `sub` claim for access token you should disable or remove `Subject (sub)` mapper.
+
+The `Subject (sub)` mapper allows you to configure the `sub` claim only for access token, lightweight access token and introspection response. IDToken and Userinfo always contain `sub` claim.
+
+The mapper has no effects for service account (as there is no user session) and `sub` claim is always added to the access token.
```

Comment:
```suggestion
The mapper has no effects for service accounts, because no user session exists, and the`sub` claim is always added to the access token.
```